src/main/java/data/DirectorTimeline.java

```diff
@@ -1,17 +1,30 @@
 package data;
 
+import java.util.ArrayList;
+
+import lombok.Getter;
+import lombok.Setter;
+
 /**
  * Created by Bart.
  * Class to store information about a directors timeline.
  */
 public class DirectorTimeline extends Timeline {
 
+    @Getter @Setter
+    private ArrayList<DirectorShot> shots;
```

Comment:
Why a setter? Will we be changing the entire `ArrayList` of shots? This looks wrong. Instead there should be a `clearShots()` method.